### ISSUE TITLE
bug: Crashlytics + Android Runtime 5.0 incompatible

### ISSUE THREAD

**Issue by DickSmith** [EddyVerbruggen/nativescript-plugin-firebase#1023]
I'm still narrowing this one down, but since the RC and even up to the full release of {N} 5.0, I have been unable to get the app to run consistently.

Essentially when live-syncing with Crashlytics enabled on the app (even with 0 actual calls to any of its methods) and when using the Android Runtime 5.0, the app will launch and then will just sit on the launch screen, with the emulator or device still expending significant processing (and getting warm).

**Comment by EddyVerbruggen** [comment]
Yeah I see a startup crash on Android with NativeScript 5.0.0 as well, when using the demo-ng app.

**Comment by DickSmith** [comment]
https://github.com/NativeScript/android-runtime/pull/1209/files

I haven't looked at it, and was about to do similar myself as previously mentioned, but this PR may be the solution.

**Comment by EddyVerbruggen** [comment]
I've just tried 3.2.1 but it doesn't help, unfortunately. Appreciate the tip though.

**Comment by NickIliev** [comment]
@EddyVerbruggen it seems that the latest android gradle plugin has changed the location of the folder that contains the metadata. The `nativescript-plugin-firebase` is manually copying the fabric gradle 
 plugin's metadata to the intermediates assets output folder. 
With the bumping of the android gradle plugin version, this location is no longer valid which is causing the crash.

**Comment by EddyVerbruggen** [comment]
Thanks a million, @NickIliev! It's indeed the exact same problem I tackled before in #549 (but totally forgot about it). That `assets` folder changed to `merge[Debug]Assets` and the script needs to adjusted accordingly. I should be able to make this compatible with any {N} version. Gonna fly SOF ✈️AMS now, but will take care of it when I'm home.

**Comment by EddyVerbruggen** [comment]
Hey @KristianDD thanks! I actually created something very similar on the flight home today. Will bump the version shortly.

**Comment by DickSmith** [comment]
@EddyVerbruggen 
Just tried the build and can confirm. 😁
